package.json

```diff
@@ -54,7 +54,7 @@
     "source-map-support": "0.5.21"
   },
   "devDependencies": {
-    "@chainlink/external-adapter-framework": "0.31.0",
+    "@chainlink/external-adapter-framework": "^0.31.0",
```

Comment:
I think the repo's package.json may have been updated when updating the adapter's framework version. Could you revert this change?

Reply:
Resolved by executing the following from `ion.au/`
`yarn add @chainlink/external-adapter-framework`

Reply:
Sorry to nitpick but could you pin this to the exact version by removing the `^`?